proto/spec/spec.proto

```diff
@@ -11,6 +11,7 @@ import "spec/service_api.proto";
 message Spec {
   string index = 1; 
   string name = 2; 
+  repeated string imports = 12;

```

Comment:
we have some conflict from main. some more fields were added to spec and the number 12 is taken :).
merge main to the branch and decide on the number, than generate the proto buffs.

Reply:
Done